jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -419,6 +418,24 @@ public boolean isOffline() {
     return session.isOffline();
   }
 
+  @VisibleForTesting
+  Path getWarArtifact() {
+    Build build = project.getBuild();
+    String warName = build.getFinalName();
+
+    Plugin warPlugin = project.getPlugin("org.apache.maven.plugins:maven-war-plugin");
```

Comment:
this wont work if they set the property on the command line? or does it?

Reply:
Is it possible to set the configuration property on the command line? Does this work for any Maven plugin?

Reply:
I'm not sure, we may not HAVE to support it, but it'd be useful to see if it's possible and document it.

Reply:
From my investigation, `warName` does not have the user property (such as `maven.compiler.release`) in recent versions. Actually, `warName` isn't documented in the [latest `maven-war-plugin` doc](https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html). Looks like it was in an [old version](https://maven.apache.org/plugins-archives/maven-war-plugin-2.4/war-mojo.html#warName), but I think now it's retained only for backward compatibility in recent versions. The doc for the old version does say the user property is `war.warName`, but setting it in a recent version does nothing.

Reply:
sgtm